packages/typespec-vscode/src/log/console-log-listener.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-console */
```

Comment:
Refactor the logging related code into the log folder. and decouple vscode from logger so that the logging can still work in unit test by using the console log listener,